src/components/include/application_manager/application_manager.h

```diff
@@ -390,6 +390,12 @@ class ApplicationManager {
   virtual bool IsAppTypeExistsInFullOrLimited(
       ApplicationConstSharedPtr app) const = 0;
 
+  /**
+   * @brief Completes application registration
+   * @param application Application which registration should be completed
```

Comment:
@IGapchuk Add more details to description. It is not clear what does it mean `Completes application registration`. But what is under the hood?